specification/codesigning/CodeSigning/main.tsp

```diff
@@ -0,0 +1,221 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-autorest";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Autorest;
+
+@useAuth(
+  OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit,
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/v2.0/authorize",
+      scopes: ["https://codesigning.azure.net/.default"],
+    }
+  ]>
+)
+@service({
+  title: "Azure.CodeSigning service",
+})
+@versioned(Azure.CodeSigning.Versions)
+@server(
+  "https://{region}.codesigning.azure.net/",
+  "Azure CodeSigning is a service that provides managed code signing for all.",
+  {
+    @doc("The region where we will sign code.")
+    region: string,
+  }
+)
+@doc("Azure CodeSigning is a service that provides managed code signing for all.")
+namespace Azure.CodeSigning;
+
+enum Versions {
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2022_06_15_preview: "2022-06-15-preview",
+}
+
+interface CertificateProfileOperations {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is a custom operation status endpoint."
```

Comment:
Why can you not use an operation template from Azure Core or Foundations? It looks like Foundations.GetOperationStatus should work.

Reply:
Sorry ... I see now that you are using GetOperationStatus. Perhaps we need to promote this into Azure Core proper.

Reply:
My assumption is that there's no action needed from my end on this finding, as of now. I'm marking as resolved, but feel free to unresolve otherwise.